armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, r, cs):
     @staticmethod
     def specifyInputs(cs):
         files = {
-            cs.getSetting(settingName).label: [
+            cs.getSetting(settingName): [
```

Comment:
I believe with this change, you have to go and change the docstring for `Interface.specifyInputs()`:

https://github.com/terrapower/armi/blob/fdc6bab730b9bb06ee78810f8f8d1693bb3ec9ea/armi/interfaces.py#L422-L423

Reply:
I agree!

Reply:
Noted on the docstring. 

I'm not super familiar with the annotation syntax, but should the return now exclude the string since the key in the dictionary is now only a setting object?

Reply:
Yes I agree with you @opotowsky. I also don't know where the list comes from. I think should only be a dict?

Reply:
The dictionary that a given `specifyInputs` method returns can have a key that is either a string or setting object, still (so my question is probably not relevant, because I was thinking it should _only_ be a setting object when I asked the question). The value is enforced as a list because there can be a list of files for some settings, I think.

Reply:
Darn, so we didn't get to this in this PR?

Okay, I will open another PR to fix it.